### DISCUSSION TITLE
[PLEASE DELETE]

### DISCUSSION THREAD

**Discussion by jenningsb2** [outline/outline#3620] [Ideas]
When I use the `find or create doc...` typeahead, it is constrained only to **page search.** I would like to be able to link to **headings _within pages_** using this type ahead. 

My current workaround is to link to the page, then manually add the heading slug. 

<img width="444" alt="CleanShot 2022-06-02 at 13 01 56@2x" src="https://user-images.githubusercontent.com/29644050/171684740-40658131-545c-43c6-91ad-50090dcdfabd.png">

<img width="451" alt="CleanShot 2022-06-02 at 13 02 03@2x" src="https://user-images.githubusercontent.com/29644050/171685074-ddebacbe-6c3c-44eb-aaa4-f38d95127e9f.png">

**Comment by Limezy** [comment]
Hi, duplicate of https://github.com/outline/outline/discussions/3562
The best is to search for similar discussions before adding a new one, since it will give more "weight" to the community demand and hopefuly have the feature coming sooner to the roadmap !

**Reply by Limezy** [reply]
And by the way, you can go faster by showing the table of content and right click to copy the full URL

**Reply by jenningsb2** [reply]
Noted! Closed
